packages/framework/src/state/screen.ts

```diff
@@ -1,28 +1,28 @@
 import { atom, useAtom, useAtomValue } from "jotai";
 import { focusAtom } from "jotai-optics";
-import { clone } from "lodash-es";
+import { clone, isEqual } from "lodash-es";
 import { useCallback } from "react";
-import isEqual from "react-fast-compare";
```

Comment:
@evshi, we need this change because react-fast-compare can't properly compare objects at deeper levels. See the screenshot where it wrongly returns true even though readyState differs between records.

![image](https://github.com/EnsembleUI/ensemble-react/assets/31651612/86773241-f70e-414b-85f4-a1ff21491e8a)